lib/ObjWriter/objwriter.cpp

```diff
@@ -559,22 +556,122 @@ extern "C" void EmitDebugFileInfo(ObjectWriter *OW, int FileId,
   }
 }
 
-// This should be invoked at the end of function code emission to flush
-// debug function info.
-extern "C" void EmitDebugFunctionInfo(ObjectWriter *OW,
-                                      const char *FunctionName,
-                                      int FunctionSize) {
+static void EmitSymRecord(MCObjectStreamer &OST, int size, int symbolKind) {
+  SymRecord Rec = {ulittle16_t(size + sizeof(ulittle16_t)),
+                   ulittle16_t(symbolKind)};
+  OST.EmitBytes(StringRef((char *)&Rec, sizeof(Rec)));
+}
+
+static void EmitVarDefRange(MCObjectStreamer &OST, const MCSymbol *Fn,
+                            LocalVariableAddrRange &Range) {
```

Comment:
I meant EmitSymRecord function which now you've fixed them all. With regard to SymbolKind, it would be better to use precise enumeration type "SymbolRecordKind" instead of int for the better type safety.